.eslintrc.js

```diff
@@ -13,6 +11,7 @@ module.exports = {
   parser: "@typescript-eslint/parser",
   rules: {
     "max-classes-per-file": 0,
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.ts", "**/*.spec.ts"]}]
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.ts", "**/*.spec.ts"]}],
+    "max-len": ["error", { "code": 150, "ignoreComments": true }],
```

Comment:
The prettier config turned this rule and some others off. The other linting errors were very small or auto fixable but this one would require a bit more attentention, so not for now.